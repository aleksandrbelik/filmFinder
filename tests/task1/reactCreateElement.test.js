import React from 'react';
import ReactCreateElement from 'Task1/reactCreateElement.jsx';
import renderer from 'react-test-renderer';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';

afterEach(cleanup)

it('renders correctly', () => {
	const tree = renderer
		.create(<ReactCreateElement />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});

it('renders a "Hello World" message', () => {
  const { container, getByText } = render(<ReactCreateElement />);
  expect(getByText('Hello World! (React Create Element component)')).toBeInTheDocument();
})