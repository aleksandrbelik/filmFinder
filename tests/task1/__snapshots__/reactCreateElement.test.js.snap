// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ff-react-create-element"
  id="ff-react-create-element-id"
>
  Hello World! (React Create Element component)
</div>
`;
